software/24.Git_GitHub/GitHub/README.md

```diff
@@ -0,0 +1,60 @@
+# Part 2: GitHub :rocket:
+
+## Step 0 - Project initialization
+Create a repository and commit the files & folders contained in [project folder].
+Some steps should be completed in groups of at least 2.
+Go to your repository settings, and in the `Collaborators` add your teammate !
+
+<i>Pro tips:</i>
+> Don't forget to use the commit norm you learned in the previous part :wink:
+
+> Don't forget to use branches ! This time, try to create it directly in GitHub :rocket:
+
+> This workshop is based on [PoC Innovation's Open-Source project template](https://github.com/PoCInnovation/open-source-project-template).  
+Several links to specific parts of the [getting started guide](https://github.com/PoCInnovation/open-source-project-template/blob/main/.github/getting-started.md) will be given to you, but don't spend too much time on it, we have a lot to cover ! You can read it entirely later.
+
+## Step 1 - [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues)
+On a project, you'll need a tool to track and organize your work: with GitHub Issues, you can track feature requests, user feedbacks and bug reports.  
+If you played a bit with the calculator you may notice the given result isn't correct.  
+You have to create 3 issues:
+- One to describe the problem, with expected behaviour and what you are experiencing.
+- Another to suggest a feature (add all the arguments instead of just the first 2)
+- A last one to take a single string as parameter and support operators in it (this one will not be implemented)
+
+Method: result screen to reproduce or fully autonomous with correction after
+
+> Take a look at [labels](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels) and [assignees](https://docs.github.com/en/issues/tracking-your-work-with-issues/assigning-issues-and-pull-requests-to-other-github-users)
+
+## Step 2 - [Pull Requests (PRs)](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests)
+Congratulations, you solved these issues !  
+But don't close them yet, you will now learn to use Pull Requests and use the option to close issues automatically :sparkles:
+
+Create 2 Pull Requests, and use 2 different methods to link the related issues.
+When it's done, add your teammate as reviewer and play with the review tool in the `Files Changed` tab !
+
```

Comment:
You also can talk about the pull requests templates and what are the best practices.